src/DebugUtil.h

```diff
@@ -41,14 +41,14 @@
 
 #define BYTETOBINARYPATTERN "%d%d%d%d%d%d%d%d"
 #define BYTETOBINARY(byte)  \
-  (byte & 0x80 ? 1 : 0), \
-  (byte & 0x40 ? 1 : 0), \
-  (byte & 0x20 ? 1 : 0), \
-  (byte & 0x10 ? 1 : 0), \
-  (byte & 0x08 ? 1 : 0), \
-  (byte & 0x04 ? 1 : 0), \
-  (byte & 0x02 ? 1 : 0), \
-  (byte & 0x01 ? 1 : 0) 
+  ((byte & 0x80) ? 1 : 0), \
+  ((byte & 0x40) ? 1 : 0), \
+  ((byte & 0x20) ? 1 : 0), \
+  ((byte & 0x10) ? 1 : 0), \
+  ((byte & 0x08) ? 1 : 0), \
+  ((byte & 0x04) ? 1 : 0), \
+  ((byte & 0x02) ? 1 : 0), \
+  ((byte & 0x01) ? 1 : 0) 
```

Comment:
The extra (...) is a bit useless, but okay. I can live with this :-)

Reply:
Style question. Not everybody is good at knowing all operator preferences for those cases by heart, it leaves room for misunderstandings.
Actually, it causes warnings in static code analysis (cppcheck --enable=all .):
[KonnektingDevice.cpp:151]: (style) Clarify calculation precedence for '&' and '?'.
[KonnektingDevice.cpp:367]: (style) Clarify calculation precedence for '&' and '?'.
...
This is why I changed it, despite being a bit useless ;-)